manifests/install.pp

```diff
@@ -1,13 +1,24 @@
 class solr5::install inherits solr5 {
-  download_uncompress { $package_url:
-    distribution_name => $package_url,
-    dest_folder => $package_target_dir,
-    creates => $sources,
-    uncompress => 'tar.gz'
+
+  include wget
```

Comment:
Because we need the archive completely unextracted. The solr installer is build that way. download_uncompress is actually leaving the archive in /tmp, but there is no guarantee for it that it stays that way.
Also I thought it would be irritating working on something which is not intentionally created.